### ISSUE TITLE
Increase limit for pinned chats

### ISSUE THREAD

**Issue by crackedmind** [procxx/tdesktop#11]

**Comment by crackedmind** [comment]
@crackedmind на подумать, пушо ограничения апи, но ябы локально в клиенте такое разрешил

**Comment by Kaffeine** [comment]
The limit is set on the server side (it comes from reply to help.getConfig())

It is still possible to pin an arbitrary number of dialog on the client-side.

**Comment by crackedmind** [comment]
@Kaffeine Yes, we know about server-side limit, and want to increase client-side limits.

**Comment by Kaffeine** [comment]
The issue is marked as a question and there was been no clarification. :-)

**Comment by berkus** [comment]
No problems, would be nice to synchronize this list between client instances of course, but we could use a side-channel for that.
